Tictactoe.cs

```diff
@@ -112,7 +134,7 @@ private bool MakeMove(string input)
 
         private bool CanTakeTurn(string input)
         {
-            var hasUserQuit = HasUserQuit(input);
+            var hasUserQuit = UserHasQuit(input);
```

Comment:
How can a player take a turn if that player has quit? Consider moving the quit check before the take turn check?